src/Parser/VengeanceDemonHunter/Modules/Statistics/Spells/SigilOfFlame.js

```diff
@@ -10,24 +13,25 @@ import { formatDuration } from 'common/format';
 import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
 
 class SigilOfFlame extends Module {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    enemies: Enemies,
+  };
 
   statistic() {
 
-    const sigilOfFlameUptime = this.owner.modules.enemies.getBuffUptime(SPELLS.SIGIL_OF_FLAME_DEBUFF.id);
+    const sigilOfFlameUptime = this.enemies.getBuffUptime(SPELLS.SIGIL_OF_FLAME_DEBUFF.id);
 
     const sigilOfFlameUptimePercentage = sigilOfFlameUptime / this.owner.fightDuration;
 
-    if(this.owner.modules.abilityTracker.abilities[SPELLS.SIGIL_OF_FLAME_DEBUFF.id]) {
-      this.sigilOfFlameDamage = this.owner.modules.abilityTracker.abilities[SPELLS.SIGIL_OF_FLAME_DEBUFF.id].damageEffective;
-    }
-
+    const sigilOfFlameDamage = this.abilityTracker.getAbility(SPELLS.SIGIL_OF_FLAME_DEBUFF.id).damageEffective;
 
     return (
       <StatisticBox
         icon={<Icon icon="ability_demonhunter_sigilofinquisition" alt="Sigil of Flame" />}
```

Comment:
Please use `SpellIcon` here to users can consistently hover the icon for tooltips about the spells to learn more about them.